README.md

```diff
@@ -1849,6 +1850,82 @@ Delete a cache cluster
 
     disco_elasticache.py [--env ENV] delete --cluster CLUSTER
 
+Elasticsearch
+-------------
+
+### Introduction
+Elasticsearch is an AWS service that is capable of indexing and analysizing large amounts of data. A typical use would be analyzing and visualizing logs from instances.
+
+Elasticsearch domains can be created using `disco_elasticsearch.py`. Each environment (VPC) can have multiple Elasticsearch domains. Currently, the domain is managed independent of VPC creation/deletion. An Elasticsearch domain has a service endpoint where we can ship logs to, and interact with it via API calls.
+
+Elasticsearch Domain Name format is:
+
+`es-<elasticsearch_name>-<environment_name>`
+
+Example:
+
+`es-logs-ci`
+
+Elasticsearch Endpoint format is:
+
+`search-<cluster_name>-<cluster_id>.<region>.es.amazonaws.com`
+
+Example:
+
+`search-ci-log-es-nkcqfivhtjxy7ssl4vrr3s5cq4.us-west-2.es.amazonaws.com`
+
+After Elasticsearch cluster has been created, a CNAME record in Route53 for the endpoint is also added.
+Route 53 CNAME format:
+
+`<elasticsearch_domain_name>.<domain_name>`
+
+NOTE: `domain_name` refers to the `default_domain_name` configured in `disco_aws.ini`
+
+Example:
+
+`ci-log-es.aws.wgen.net`
```

Comment:
Arguably, our config and documentation don't need to agree. The documentation just needs to be internally consistent. I'll change all of the examples to be something else to make that more obvious. And remove the `wgen` reference. : )